crates/top/rerun-cli/Cargo.toml

```diff
@@ -45,7 +45,7 @@ default = ["native_viewer", "web_viewer"]
 
 ## The features we enable when we build the pre-built binaries during our releases.
 ## This may enable features that require extra build tools that not everyone heas.
-release = ["default", "nasm"]
+release = ["default", "nasm", "map_view"]
```

Comment:
We should also add the new feature to 

https://github.com/grtlr/rerun/blob/1408badf5522eddeae82e2ee069771e3510cf7e3/rerun_py/Cargo.toml#L20-L22

Reply:
On second thought: I don't think so?

That feature only controls whether or not the `re_space_view_map` crate is included, and the corresponding view registered. All the data/blueprint archetypes/components are unconditionally included regardless of the cargo feature.

This solves the dependency problem, but doesn't "hide" everything genospatial. Doing so would require way more work, incl. on codegen, so I'd rather avoid that.

With that being said, `rerun_py/Cargo.toml` doesn't depend at all on 
`re_space_view_map`, `re_viewer`, `rerun` or `rerun_cli` (which are affected by that feature).